service/src/main/kotlin/fi/espoo/evaka/serviceneed/ServiceNeed.kt

```diff
@@ -47,10 +47,10 @@ data class ServiceNeedOptionSummary(
 )
 
 data class ServiceNeedConfirmation(
-    val employeeId: UUID,
-    val firstName: String,
-    val lastName: String,
-    val at: HelsinkiDateTime
+    val employeeId: UUID?,
```

Comment:
I recall seeing an annotation @PropagateNull, which could be used to make `val confirmed: ServiceNeedConfirmation` to be null instead of having these fields be separately nullable.

Reply:
e.g. here https://github.com/espoon-voltti/evaka/blob/master/service/src/main/kotlin/fi/espoo/evaka/messaging/message/Message.kt#L70

Reply:
How to achieve same with json-query, like https://github.com/espoon-voltti/evaka/blob/master/service/src/main/kotlin/fi/espoo/evaka/placement/PlacementService.kt#L409-L414 ?

Reply:
Fixed in https://github.com/espoon-voltti/evaka/pull/1280/commits/fc279664a0c806ff732401c8659afc08a284e93f